setup.cfg

```diff
@@ -0,0 +1,57 @@
+[metadata]
+name = create-aio-app
+version = attr: create_aio_app.utils.versiontools.get_self_version
+description = The tool that helps quickly create a basis for
```

Comment:
The description should be short, I'm almost sure that you cannot put new lines in it.

Reply:
Yes, I checked, more than one line spoils formatting on pypi.
How about "The tool that helps quickly create a basis for aiohttp application"?

Reply:
It's not because of PyPI, it is because of invalid metadata

Reply:
@Arfey ^